### DISCUSSION TITLE
[Feature] Add more ITCase for log remote storage

### DISCUSSION THREAD

**Discussion by swuferhong** [apache/fluss#92] [General]
### Search before asking

- [X] I searched in the [issues](https://github.com/alibaba/fluss/issues) and found nothing similar.

### Motivation

Fluss remote log storage currently only includes a few tests, Many operations, such as the creation, upload, download, and metadata update of remote logs lacking necessary tests. You can add more tests in `RemoteLogITCase` to improve the testing process.

### Willingness to contribute

- [ ] I'm willing to submit a PR!

**Comment by hunter-cloud09** [comment]
In order to get familiar with the project, I will complete this issue, please assign it to me. thx!

**Comment by wuchong** [comment]
convert to issue #137
